mods/default/locale/default.fr.tr

```diff
@@ -0,0 +1,211 @@
+# textdomain: default
+Locked Chest=Coffre verrouillé
+Locked Chest (owned by @1)=Coffre verrouillé (possédé par @1)
+You do not own this chest.=Ce coffre ne vous appartient pas.
+a locked chest=Un coffre verrouillé
+Chest=Coffre
+Stick=Baton
+Paper=Papier
+"@1" by @2=« @1 » de @2
+Book=Livre
+Book with Text=Livre avec du texte
+Skeleton Key=Squelette
+Key to @1's @2=Clé pour @2 de @1
+Coal Lump=Morceau de charbon
+Iron Lump=Morceau de fer
+Copper Lump=Morceau de cuivre
+Tin Lump=Morceau d'étain
+Mese Crystal=Cristal de mese
+Gold Lump=Morceau d'or
+Diamond=Diamant
+Clay Lump=Morceau d'argile
+Steel Ingot=Lingot d'acier
+Copper Ingot=Lingot de cuivre
+Tin Ingot=Lingot d'étain
+Bronze Ingot=Lingot de bronze
+Gold Ingot=Lingot d'or
+Mese Crystal Fragment=Fragment de cristal de mese
```

Comment:
I think the consensus is to use "Mese" as a proper noun, which means it should be capitalized.

Make sure to apply this change in all strings where "mese" is written.